vmaas/vulnerabilities.go

```diff
@@ -376,6 +379,6 @@ func updateCves(cves map[string]VulnerabilityDetail, cve, pkg string, erratum *s
 	vulnDetail := cves[cve]
 	vulnDetail.Packages = append(vulnDetail.Packages, pkg)
 	if erratum != nil {
-		vulnDetail.Errata = append(vulnDetail.Errata, *erratum)
+		vulnDetail.Errata = append(vulnDetail.Errata, erratum...)
```

Comment:
do we need to check for `erratum != nil`? I don't know how `erratum...` would work with empty `[]string`
If that's needed, wouldn't it better to check it like `len(erratum) > 0`? Above we are creating the slice as `errataNames := make([]string, 0)` so that's never `nil`

Reply:
Yes, with slices we should be safe without this check at all

Reply:
Spread operator is safe on nil and empty slice